torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -306,6 +306,21 @@ std::vector<at::Tensor> FusionExecutorCache::runFusionWithInputs(
     const at::ArrayRef<IValue>& inputs) {
   FUSER_PERF_SCOPE("runFusionWithInputs");
 
+  auto detect_normalization_fusion = [&]() {
```

Comment:
Since we now have normalization type merged in, can we update this implementation to avoid pattern matching?